### DISCUSSION TITLE
我还是想询问一下predict所需要的数据应该如何传入？

### DISCUSSION THREAD

**Discussion by leofionn** [Congyuwang/AlphaNetV3#17] [Q&A] [answered]
作者你好，很抱歉再次打扰。我看了之前的issues，也有问到如何使用model.predict。我尝试了一下，发现还是有一些问题。故来询问。
我的demo例子是假设我现在有000723.SZ的三十条数据，我需要如何导入测试？
![image](https://user-images.githubusercontent.com/33302972/141879910-ace5dcd9-6762-4ecf-9404-1f86784b572f.png)
我使用你说的两种方法，但是发现都有报错。
能否麻烦你帮忙给一个使用小例子？

**Comment by Congyuwang** [comment, marked as answer]
https://github.com/Congyuwang/AlphaNetV3/discussions/13
